### PR TITLE
(ietester) Updated + Moved to manual update

### PR DESCRIPTION
**Pull request by AdmiringWorm**
Requires `[PUSH ietester]` on merge.

### CODE REVIEW

File: manual/ietester/ietester.nuspec [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+    <docsUrl>http://www.my-debugbar.com/wiki/IETester/Doc</docsUrl>
+    <mailingListUrl>http://www.my-debugbar.com/forum/</mailingListUrl>
+    <tags>ietester internet explorer tester admin freeware</tags>
+    <summary>Browser Compatibility Check for Internet Explorer Versions from 5.5 to 11</summary>
     <description>
-IETester is a free (both for personal and professional usage) WebBrowser that allows you to have the rendering and javascript engines of IE10, IE9, IE8, IE7 IE 6 and IE5.5 on Windows 8 desktop, Windows 7, Vista and XP, as well as the installed IE in the same process.
+IETester is a free (both for personal and professional usage) WebBrowser that allows you to have the rendering and javascript engines of IE11, IE10, IE9, IE8, IE7 IE 6 and IE5.5 on Windows 8 desktop, Windows 7, Vista and XP, as well as the installed IE in the same process.
+
+## Notes
+- See the home page for known problems and limitations
```
Comment (pascalberger) [inline manual/ietester/ietester.nuspec]: "home page" doesn't make much sense in the context of a chocolatey package. I would suggest:

```
See the [website](http://www.my-debugbar.com/wiki/IETester/HomePage) for known problems and limitations.
```

Comment (AdmiringWorm) [inline manual/ietester/ietester.nuspec]: Was sure I had added the link, guess not.
Thanks, I'll fix it shortly.

**Comment by AdmiringWorm** [comment]
@pascalberger I've updated the home page note

**Comment by pascalberger** [comment]
@AdmiringWorm LGTM

**Pull request merged**